app/checks/free_ram.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import re
+from os import environ
+from sys import stderr
+
+if 'RAMPERCENT' not in environ:
+    print("Missing RAMPERCENT parameter.\nUsage ./free_ram.py env variables are RAMPERCENT, [MOCK_FREE_RAM] (in MiB)", file=stderr)
```

Comment:
I would remove MOCK_FREE_RAM from official documentation, it's for development and testing purposes only.